package.json

```diff
@@ -139,10 +139,10 @@
     "@types/amplitude-js": "8.16.5",
     "@types/audioworklet": "0.0.29",
     "@types/dom-screen-wake-lock": "1.0.1",
-    "@types/jasmine": "5.1.4",
     "@types/js-md5": "0.4.3",
     "@types/jsonwebtoken": "9.0.7",
     "@types/lodash-es": "4.17.12",
+    "@types/mocha": "10.0.10",
```

Comment:
Why the change from jasmine to mocha?

Reply:
The skip method impl is missing in jasmine. And they don't plan to add it, there was an issue about it. 
I tried using what they advice to use (pending) but I couldn't make it work and to be as elegant as it is with mocha.

Reply:
Kewl!